packages/@1/ui/react/src/grid/atom.ts

```diff
@@ -6,6 +6,8 @@ import { tv } from "tailwind-variants";
 
 export const column_screen = tv({
   base: `
+    max-h-[calc(100vh_-_theme(spacing.24
+    ))]
```

Comment:
```suggestion
    max-h-[calc(100vh_-_theme(spacing.24))]
```

Will need to reorder with prettier.
Might be in conflict with `max-h-[calc(100vh_-_theme(spacing.32))]`